rsl_rl/runners/on_policy_runner.py

```diff
@@ -42,8 +42,12 @@ def __init__(self, env: VecEnv, train_cfg, log_dir=None, device="cpu"):
         self.save_interval = self.cfg["save_interval"]
         self.empirical_normalization = self.cfg["empirical_normalization"]
         if self.empirical_normalization:
-            self.obs_normalizer = EmpiricalNormalization(shape=[num_obs], until=1.0e8).to(self.device)
-            self.critic_obs_normalizer = EmpiricalNormalization(shape=[num_critic_obs], until=1.0e8).to(self.device)
+            if train_cfg.get("resume") == True:
+                until = 0
+            else:
+                until = 1.0e8
+            self.obs_normalizer = EmpiricalNormalization(shape=[num_obs], until=until).to(self.device)
+            self.critic_obs_normalizer = EmpiricalNormalization(shape=[num_critic_obs], until=until).to(self.device)
```

Comment:
I think the correct way to solve the problem is to save and load the internal count of EmpiricalNormalization. The current solution breaks if you resume training before 1.0e8 steps

Reply:
https://github.com/tasdep/rsl_rl-1/blob/f2b721aae7c33a34155a0db6135a559b75513334/rsl_rl/runners/on_policy_runner.py#L257-L262
what is saved_dict[infos] intended for? Or should I just add another term to the saved_dict?